requirements.txt

```diff
@@ -24,4 +24,4 @@ transformers
 s3fs
 wikipedia
 ipython
-llm-foundry>=0.4.0
+# llm-foundry>=0.4.0
```

Comment:
agree its ok to remove this as a dependency. can we guard the `import` here with a try/catch telling the user to pip install or pointing to the README? : https://github.com/mlfoundations/open_lm/blob/6bb1247d8e668099fdfcacbe6a42d4f3ba87d5cc/eval/eval_openlm_ckpt.py#L10